Game2048Base/src/main/java/models/square/Square.java

```diff
@@ -11,7 +11,6 @@
     static final double MEDIUM_VALUE_TEXT_SIZE_OFFSET = .35;
     static final double LARGE_VALUE_TEXT_SIZE_OFFSET = .3;
     static final double VERY_LARGE_VALUE_TEXT_SIZE_OFFSET = .3;
-
```

Comment:
I actually think this space between constants and instance vars was good and would leave it.